website/content/docs/concepts/domain-model/groups.mdx

```diff
@@ -51,3 +51,7 @@ A group has the following configurable attributes:
 The following services are relevant to this resource:
 
 - [Group Service](/api-docs/group-service)
+
+## Tutorial
+
+Refer to the [Manage Users and Groups with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-users-groups?in=boundary/hcp-administration) tutorial to learn how to complete user managemement related tasks.
```

Comment:
for all tutorial links, I like to remove the collection URL in order to ensure the tutorial can be found if it's moved to another collection, or if a redirect is created. I suffered this recently 

```suggestion

## Tutorial

Refer to the [Manage Users and Groups with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-users-groups) tutorial to learn how to complete user management related tasks.
```
```suggestion

## Tutorial

Refer to the [Manage Users and Groups with HCP Boundary](https://learn.hashicorp.com/tutorials/boundary/hcp-manage-users-groups?in=boundary/hcp-administration) tutorial to learn how to complete user managemement related tasks.
```